codegen/templates/main.tmpl

```diff
@@ -29,16 +30,22 @@ func getConfigDirName() string {
 	)
 }
 
-func getConfig() *zanzibar.StaticConfig {
+func getConfig(seedConfig map[string]interface{}) *zanzibar.StaticConfig {
 	return zanzibar.NewStaticConfigOrDie([]string{
 		filepath.Join(getDirName(), "zanzibar-defaults.json"),
 		filepath.Join(getConfigDirName(), "production.json"),
 		filepath.Join(os.Getenv("CONFIG_DIR"), "production.json"),
-	}, nil)
+	}, seedConfig)
 }
 
 func createGateway() (*zanzibar.Gateway, error) {
-	config := getConfig()
+    listenIP, err := tchannel.ListenIP()
```

Comment:
We want to listen on both IPs.

Listening on just one IP is going to make local development frustrating, as well as authoring tests frustrationg.